### ISSUE TITLE
Wrong answer from tflite model with certain configurations of depthwise conv2d

### ISSUE THREAD

**Issue by boweia** [tensorflow/tensorflow#61815]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Debian 11
- TensorFlow installation (pip package or built from source): pip package
- TensorFlow library (version, if pip package or github SHA, if built from source): 2.13.0

### 2. Code

```
import tensorflow as tf
import numpy as np

tf.keras.utils.set_random_seed(0)

# Layer parameter values
imageSize = (8, 8)
numChannels = 2
fmt = 'channels_last'
kernelSize = (3, 1)
padding = 'same'
depthMult = 1
useBias = False
strides = (2, 2)

# Determine input shape
if fmt == 'channels_last':
    inShape = imageSize + (numChannels,)
else:
    inShape = (numChannels,) + imageSize

# Determine input size and create input data
inSize = (1,) + inShape
rng = np.random.default_rng(seed=999)
x = rng.random(size=inSize, dtype=np.float32)

# Construct Keras model
inp = tf.keras.layers.Input(shape=inShape, batch_size=1)
dconv = tf.keras.layers.DepthwiseConv2D(kernelSize, strides=strides, padding=padding, depth_multiplier=depthMult,
                                        data_format=fmt, activation=None, use_bias=useBias)(inp)
kModel = tf.keras.models.Model(inputs=inp, outputs=dconv)
kModel.compile()

# Run Keras model predict
kY = kModel.predict(x)
print(kY)

# Convert model to tflite
converter = tf.lite.TFLiteConverter.from_keras_model(kModel)
tfModel = converter.convert()

# Run tflite interpreter
interpreter = tf.lite.Interpreter(model_content=tfModel)
interpreter.allocate_tensors()
input_details = interpreter.get_input_details()
output_details = interpreter.get_output_details()
interpreter.set_tensor(input_details[0]['index'], x)
interpreter.invoke()
tfY = interpreter.get_tensor(output_details[0]['index'])
print(tfY)

isequal = np.allclose(kY, tfY, rtol=1e-4, atol=1e-4)
print(isequal)
```

### 3. Failure after conversion
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results - in this case all 0s

5. (optional) Any other info / logs

Some general observations on when this wrong answer occurs (all of the following must be met):
- One of the kernel_size dimensions is 1 (1xN or Nx1)
- Stride is not 1
- Padding is same

**Comment by pkgoogle** [comment]
I am able to reproduce on nightly: [gist](https://colab.sandbox.google.com/gist/pkgoogle/11b39c4c2962fdcedce78dac2be1a1fa/61815.ipynb), The problem is the output tensor (in the TFLite case) is all zeroes.

Hi @majiddadashi, can you please take a look? Thanks.

**Comment by Venkat6871** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The Tensorflow team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the Tensorflow space.
